### ISSUE TITLE
Feature request: suppress exit status message with -q

### ISSUE THREAD

**Issue by DHMike57** [eradman/entr#137]
`entr` is a great fit for activating a test runner when either the source file or the test file changes: I have a script that sets it up
finishing like this:
`ls  $CHAPTER $TESTS | entr -psq 'guile -L . -L ~/.guile.lib/ --no-auto-compile tests/$(basename $0)'`
in a separate terminal.  I think I have to use the -s option so that I can refer both to source and test files at the same time,
but the exit message following every file change is mildly distracting. 

I patched it for my own use, but submitted a pr #136 since it would be nice to hand back updating to pacman.

**Comment by eradman** [comment]
It is true that you need to use `-s` for `$0` to be set, but `entr` cannot be used to reliably used to compile files this way. `/_` or `$0` is just the "leading edge", the file that first triggered an event, all other events are discarded.

The intent was to provide an easy way of watching a single file. From the man page:

```

     Clear the screen and run a query after the SQL script is updated:

           $ echo my.sql | entr -cp psql -f /_
```

Since this comes up periodically I added a *Processing Files* section at https://eradman.com/entrproject/

The short version is: use a built tool, and use `entr` to kick it off

```
ls  $FILES | entr -p make
```

Let me know if this make sense.

**Comment by eradman** [comment]
Got it..you don't expect "all files to be processed" when changing branches, for example, you're just editing one at a time.

Changing the Tmux background color is a cool idea!

**Comment by eradman** [comment]
I proposed a change that introduces an environment variable enable a custom exit status message: https://github.com/eradman/entr/pull/138

Hence add this to your `.profile` to disable the status message
```
export ENTR_STATUS_COMMAND=""
```

Another example:

```
export ENTR_STATUS_COMMAND='echo "== $EXIT_STATUS =="'
```

Conditional messages

```
export ENTR_STATUS_COMMAND='[ "$EXIT_STATUS" -eq 0 ] && printf "\xE2\x9C\x94 pass\n" || printf "\xE2\x9D\x8C fail\n"'
```

**Comment by eradman** [comment]
CC @dboehmer

**Comment by eradman** [comment]
https://github.com/eradman/entr/pull/141 takes an entirely different approach:

_no status_ message by default, new `-x` option for enabling status messages that are configurable

**Comment by DHMike57** [comment]
> #141 takes an entirely different approach:
> 
> _no status_ message by default, new `-x` option for enabling status messages that are configurable

This looks like a great solution for everyone.  Default does nothing suits me best, but the flexibility for future ideas is there.

**Comment by eradman** [comment]
Yeah, I think people will like quiet by default with the option to customize the status message. Merged for further testing!
